release/models/aft/openconfig-aft-common.yang

```diff
@@ -244,11 +258,156 @@ submodule openconfig-aft-common {
           }
         }
 
+        container encap-headers {
+          description
+            "Container for packet encapsulation headers.  When present, this
```

Comment:
Can we reword here -- the presence of the container (in a YANG sense) does not reflect anything. Something like "when options within this container are specified, this indicates that ...".

Reply:
Done